ecs-web/elb.tf

```diff
@@ -49,3 +49,23 @@ resource "aws_security_group_rule" "sg_ecs_instances_elb_out_ssh" {
   protocol          = "tcp"
   cidr_blocks       = ["0.0.0.0/0"]
 }
+
+# Allow traffic to Prometheus metrics from the ECS security group
+resource "aws_security_group_rule" "sg_ecs_instances_out_metrics" {
```

Comment:
I think you need to allow access to the prometheus port from the ELB no? As prometheus works by polling its targets, it'll be prometheus that makes requests to the ELB, and the ELB to the backend